adventureGame/AdventureGame/src/Main.py

```diff
@@ -125,4 +129,63 @@ def constructDungeonRoom():
         # Print for debug.
         dungeonRoom1.printDungeonRoomInfo()
 
```

Comment:
Can you add a summary of what the functions do. e.g.

\# Summary:
\#     Does blah blah

if there's a parameter:
\# @Param:
\#     ParameterName - this parameter is for blah blah